doc/releases/changelog-0.37.0.md

```diff
@@ -4,175 +4,422 @@
 
 <h3>New features since last release</h3>
 
-<h4>Execute faster with Default Tensor 🔗</h4>
+<h4>Execute wide circuits with Default Tensor 🔗</h4>
 
-* The `default.tensor` device now supports the `tn` method to simulate quantum circuits using exact tensor networks.
+* A new `default.tensor` device is now available for performing
+  [tensor network](https://en.wikipedia.org/wiki/Tensor_network) and
+  [matrix product state](https://en.wikipedia.org/wiki/Matrix_product_state) simulations
+  of quantum circuits using the
+  [quimb backend](https://quimb.readthedocs.io/en/latest/).
+  [(#5699)](https://github.com/PennyLaneAI/pennylane/pull/5699)
+  [(#5744)](https://github.com/PennyLaneAI/pennylane/pull/5744)
   [(#5786)](https://github.com/PennyLaneAI/pennylane/pull/5786)
+  [(#5795)](https://github.com/PennyLaneAI/pennylane/pull/5795)
 
-* The `default.tensor` device is introduced to perform tensor network simulations of quantum circuits using the `mps` (Matrix Product State) method.
-  [(#5699)](https://github.com/PennyLaneAI/pennylane/pull/5699)
+  Either method can be selected when instantiating the `default.tensor` device by setting the
+  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
+  device can simulate a large number of qubits. Take this example that
+  simulates 1000 qubits with a shallow circuit in a few seconds!
 
-* The wires for the `default.tensor` device are selected at runtime if they are not provided by user.
-  [(#5744)](https://github.com/PennyLaneAI/pennylane/pull/5744)
+  ```python
+  import pennylane as qml
 
-* Add operation and measurement specific routines in `default.tensor` to improve scalability.
-  [(#5795)](https://github.com/PennyLaneAI/pennylane/pull/5795)
+  num_qubits = 1000
+  dev = qml.device("default.tensor", method="mps")
+
+  @qml.qnode(dev)
+  def circuit():
+      qml.Hadamard(0)
+      for i in range(num_qubits - 1):
+          qml.CNOT([i, i + 1])
+      return qml.expval(qml.Z(num_qubits - 1))
+  ```
+
+  ```pycon
+  >>> circuit()
+  tensor(0., requires_grad=True)
+  ```
+
+  For matrix product state simulations (`method="mps"`), we can make the exeuction approximate by setting `max_bond_dim` (see the
+  [device's documentation](https://docs.pennylane.ai/en/stable/code/api/pennylane.devices.default_tensor.DefaultTensor.html)
+  for more details).
+  The maximum bond dimension lets us control the degree of the approximation and has implications for the speed of the simulation, as shown in the
```

Comment:
Just rewording a bit because we don't show any timing results

```suggestion
  The maximum bond dimension has implications for the speed of the simulation and lets us control the degree of the approximation, as shown in the
```